### PR TITLE
Updated Selection_list

### PR DESCRIPTION
**Pull request by genusistimelord**
I have 

- updated the SelectionList.
- Cleaned up warnings
- Fixed the unneeded 'a error.

### CODE REVIEW

File: src/native/selection_list.rs [hunk 1]
```diff
@@ -23,7 +27,7 @@ where
     /// Label Font
     font: Renderer::Font,
     /// Style for Looks
-    style: selection_list::Style,
+    style: Style,
```
Comment (Kaiden42) [inline src/native/selection_list.rs:30]: shouldn't this be `style_sheet: Box<dyn StyleSheet + 'a>,`?

File: src/native/selection_list.rs [hunk 2]
```diff
@@ -203,40 +217,24 @@ where
             &layout.bounds(),
         );
 
-        renderer.decorate(layout.bounds(), cursor_position, &self.style, primitives)
+        renderer.fill_quad(
```
Comment (Kaiden42) [inline src/native/selection_list.rs]: You are drawing the background after the content but it should be drawn first.

File: src/native/selection_list.rs (hunk 1 above)
Comment (genusistimelord) [reply src/native/selection_list.rs:30]: sadly Selection list never used the stylesheet and only the style directly. IDR why we decided to do it this way but it was due to some reason. maybe a lifetime issue.

**Review by Kaiden42 — CHANGES_REQUESTED** [review]
Looks good, but small changes requested.

File: src/native/selection_list.rs (hunk 1 above)
Comment (Kaiden42) [reply src/native/selection_list.rs:30]: okay, than keep it that way :+1:

**Comment by genusistimelord** [comment]
changes done.

**Review by Kaiden42 — APPROVED** [review]
Great! Thank you very much!

**Pull request merged**